### DISCUSSION TITLE
The difference between favoritor_score vs favoritor_total?

### DISCUSSION THREAD

**Discussion by fongfan999** [jonhue/acts_as_favoritor#274] [Q&A] [answered]
Hello there, 

I'd like to know the difference between **favoritor_score** vs **favoritor_total**? I looked at my database and they were the same all the time. Could you please help me to give me a few examples that would generate different values?

Thank you so much for your contribution.

**Comment by jonhue** [comment, marked as answer]
From the readme:

> The total counts all favorites that were recorded, while the score factors in favorites that were removed. In most use cases the score is the most useful.

There is also this previous discussion on the same topic: https://github.com/jonhue/acts_as_favoritor/discussions/133

Let me know if you have any further questions/anything is unclear!

**Reply by fongfan999** [reply]
Thank you so much for your prompt reply.

I didn't aware that there's a **Discussions** section on the repo, so I didn't search the issue first before asking, I'm so sorry about that.

**Reply by jonhue** [reply]
No worries! It happens to the best of us :)
